repositories/favoriteList.js

```diff
@@ -15,7 +15,19 @@ favoriteListSchema.add({
   restaurants: [restaurantSchema]
 })
 
-favoriteListSchema.method('toJSON', modelHelpers.toJSON)
+favoriteListSchema.methods.toJSON = function () {
+  const obj = modelHelpers.toJSON()
```

Comment:
Can't use modelHelpers.toJSON this way without redefining the `this` otherwise it crashes -- Using something like `const obj = modelHelpers.toJSON.call(this)` fixes it